YANG/ccamp/l1csm/ietf-l1csm.yang

```diff
@@ -132,7 +132,7 @@ module ietf-l1csm {
         type string;
         must '. != ../../endpoint-1/uni' {
           error-message
-            "The two end points must not be equal to each other. ";
+            "The two end point IDs must not be equal to each other. ";
```

Comment:
```suggestion
            "The two end point IDs must not be equal to each other.";
```

The line exceeds the 69 character limitation (just by 1 character) and there is no need to have the extra whitespace.